### PR TITLE
chore(about): added NPadrutt as a contributor in the About screen

### PR DESCRIPTION
**Pull request by gep13**

### CODE REVIEW

File: Gitter/Gitter/Gitter.Shared/ViewModel/Concrete/AboutViewModel.cs [hunk 1]
```diff
@@ -110,6 +110,13 @@ public string ApplicationVersion
                 Image = "https://avatars.githubusercontent.com/u/5465074",
                 Contribution = "Contributor"
             }
+            ,
```
Comment (Odonno) [inline Gitter/Gitter/Gitter.Shared/ViewModel/Concrete/AboutViewModel.cs]: Fix typo `},`

**Comment by Odonno** [comment]
Sure !

**Comment by NPadrutt** [comment]
oh, thank you =)

File: Gitter/Gitter/Gitter.Shared/ViewModel/Concrete/AboutViewModel.cs (hunk 1 above)
Comment (gep13) [inline Gitter/Gitter/Gitter.Shared/ViewModel/Concrete/AboutViewModel.cs]: oops, sorry, didn't see that, two mins...

Comment (gep13) [inline Gitter/Gitter/Gitter.Shared/ViewModel/Concrete/AboutViewModel.cs]: Done

**Comment by Odonno** [comment]
@gep13 @NPadrutt Perfect !

**Pull request merged**